lib/ref_arch_setup/bolt_helper.rb

```diff
@@ -60,6 +62,21 @@ def self.run_task_with_bolt(params)
       return success
     end
 
+    # Convert params to string for bolt
+    # format is space separated list of name=value
+    #
+    # @author Randell Pelak
+    #
+    # @param params [Array] params to convert
+    #
+    # @return [String] stringified params
+    def self.params_to_string(params)
+      str = ""
+      params.each { |k, v| str << k << "=" << v << " " }
+      str.strip!
```

Comment:
I think you can just do:
params.map{|k,v| "#{k}=#{v}"}.join(" ")

Reply:
sweet, I figured something like this was possible.  Just didn't know it.